### DISCUSSION TITLE
Is all the data you see in the Chia GUI stored in the local SQLite database file?Can I get data from the GUI interface by querying the data in SQLite?

### DISCUSSION THREAD

**Discussion by shendejun** [Chia-Network/chia-blockchain#6034] [Support]
Is all the data you see in the Chia GUI stored in the local SQLite database file?Can I get data from the GUI interface by querying the data in SQLite?

**Comment by tiny0little** [comment]
i think sqlite db stores

- blockchain in blockchain_v1_mainnet.sqlite
- peer connections in peer_table_node.sqlite
- wallet db in blockchain_wallet_v1_mainnet_*.sqlite
- and wallet peer connections in wallet_peers.sqlite

but not data for GUI

some of it stored in ~/.chia/mainnet/config/config.yaml

**Comment by BrandtH22** [comment]
@shendejun , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
